### DISCUSSION TITLE
Setting up a Canned Message greeting the user with only his first name, like an {lhc.nick}.split(" ")[0] ?

### DISCUSSION THREAD

**Discussion by fabioascoelho** [LiveHelperChat/livehelperchat#2100] [Q&A]
Can i create a new variable or modify the {args.chat.nick} or {lhc.nick} or {nick} to split the user input name and return just the first word?
If the input name is "John Doe" I want to create a Canned Message to say: "Hi, John" and not "Hi, John Doe"
But i don't know where or how to apply this, it would be nice if i could use an {lhc.nick}.split(" ")[0] 😊

**Comment by remdex** [comment]
you can listen in your exyension for this event 'chat.dynamic_array' just listen in your extension. Ise github search to find what it passes. {args.chat.dynamic... just explore chat object from audit window there is tool for that :)
